hoodie-cli/hoodie-cli.sh

```diff
@@ -9,4 +11,8 @@ if [ -z "$SPARK_CONF_DIR" ]; then
   echo "setting spark conf dir"
   SPARK_CONF_DIR="/etc/spark/conf"
 fi
-java -cp ${HADOOP_CONF_DIR}:${SPARK_CONF_DIR}:$DIR/target/lib/*:$HOODIE_JAR org.springframework.shell.Bootstrap
+if [ -n "$1" ]; then
+  echo "setting client jar"
```

Comment:
whats a client jar? is this same as hoodie jar? can we take that also from a env variable like above and not as an argument?

Reply:
So this client jar is the jar using hoodie, basically the ingestion job jar that has the Merge code.

Reply:
lets make that into a env variable then, not $1

Reply:
@n3nash I think this is the only pending item here..